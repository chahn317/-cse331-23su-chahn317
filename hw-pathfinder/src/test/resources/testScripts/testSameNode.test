# Test case where there is a trivial path from a node to itself

CreateGraph graph1

AddNode graph1 A
AddNode graph1 B

FindPath graph1 A A

FindPath graph1 B B