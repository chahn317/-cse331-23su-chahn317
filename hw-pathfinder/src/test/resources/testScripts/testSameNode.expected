# Test case where there is a trivial path from a node to itself

created graph graph1

added node A to graph1
added node B to graph1

path from A to A:
total cost: 0.000

path from B to B:
total cost: 0.000